server/repository/odpDraft/odpDraftRepository.ts

```diff
@@ -1,14 +1,12 @@
 import { OdpClassRepository } from '@server/repository'
-import { insertAudit } from '../audit/auditRepository'
 
 export const getDraftId = async (client: any, odpId: any) => {
-  const res = await client.query('SELECT draft_id FROM odp WHERE id = $1', [odpId])
-  return res.rows[0].draft_id
+  const [{ draft_id: draftId }] = await client.query('SELECT draft_id FROM odp WHERE id = $1', [odpId])
+  return draftId
 }
 
 export const updateDraft = async (client: any, draft: any) => {
```

Comment:
this belong to service. one repository cannot import another repository

Reply:
separate pr

Reply:
why?